docs/user-manuals/advanceddaemonset.md

```diff
@@ -217,6 +225,20 @@ metadata:
     lifecycle.apps.kruise.io/state: PreparingDelete     # so we update it to `PreparingDelete` state and wait for user controller to do something and remove the label
 ```
 
+#### MarkPodNotReady
+**FEATURE STATE:** Kruise v1.2.0
+
+```yaml
+  lifecycle:
+    preDelete:
+      markPodNotReady: true
+      finalizersHandler:
+      - example.io/unready-blocker
+```
+If you set `markPodNotReady=true` for `preDelete`, Kruise will try to set `KruisePodReady` condition to `False` when Pods enter `PreparingDelete` lifecycle state, and Pods will be **NotReady**, but containers still `Running`.
```

Comment:
plz add comments about the use case, such as
One can use markPodNotReady to drain service traffic before terminate containers.